features/support/hooks.rb

```diff
@@ -118,11 +118,12 @@
   line_number = scenario.location.line.to_s
 
   # Network logs
-  if page.driver.browser.respond_to?(:manage)
+  if page.driver.browser.respond_to?(:logs)
```

Comment:
this is a change in selenium I guess?

Reply:
Yeah, I actually found this in an old branch - I guess the API changed between versions https://github.com/3scale/porta/commit/a4e8de59f68f6c9bf49fb35cafcdc1e471dab0b0